skimage/morphology/tests/test_skeletonize.py

```diff
@@ -145,6 +145,14 @@ def test_zeros(self):
         image = np.zeros((10, 10), dtype=bool)
         assert np.all(thin(image) == False)
 
+    @pytest.mark.parametrize("dtype", [bool, float, int])
+    def test_thin_copies_input(self, dtype):
+        """Thin mustn't modify the original input image."""
```

Comment:
```suggestion
        """Ensure thinning does not modify the input image."""
```
or "... the original image."